<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>종합 평점</string>
  </property>
  <widget class="QFrame" name="mainframe">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>700</width>
     <height>530</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="to_mainButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
     </font>
    </property>
    <property name="text">
     <string>돌아가기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gpaConvertButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
     </font>
    </property>
    <property name="toolTip">
     <string>GPA를 4.5점/100점 만점으로 변환합니다.</string>
    </property>
    <property name="text">
     <string>GPA 변환</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>누적GPA(열람)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>누적GPA(반영)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelA1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>◆ 현재 평점</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="copyButtonA">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>15</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>복사</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>누적GPA(열람)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>누적GPA(반영)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="copyButtonB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>15</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>복사</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelA2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>◆ 가정 반영</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_arrow">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>95</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>461</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label10">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label15">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>D+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>A+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit7">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label11">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>B+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label13">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>C+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelA3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>360</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>◆ 앞으로 받을 평점별 학점 가정(F 및 재수강 없음)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label16">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label14">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>72</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>초기화</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame3_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelA4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>◆ 총평균평점 목표</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>앞으로의 평점평균 </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelA5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>◆ 수강 가능 학점</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label18">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>학점</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label20">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>201</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>이상을 달성해야 목표 도달 가능</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>열람</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>반영</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="calculateButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>45</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>계산</string>
     </property>
    </widget>
    <widget class="QLabel" name="label17">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>/4.5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
